easybuild/easyconfigs/r/Rmath/Rmath-3.3.1-intel-2016b.eb

```diff
@@ -0,0 +1,49 @@
+# Easyconfig for Rmath
+# Author: Caspar van Leeuwen
+# SURFsara, Amsterdam, The Netherlands
+
+easyblock = 'ConfigureMake'
+
+name = 'Rmath'
+version = '3.3.1'
+
+homepage = "https://www.r-project.org/"
+description = """Rmath is the standalone version of the R math library.
+ Rmath can be used in your own C/C++ routines."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+sources = ['R-%(version)s.tar.gz']
+source_urls = ['http://cran.us.r-project.org/src/base/R-%(version_major)s/']
+checksums = ['f50a659738b73036e2f5635adbd229c5']
+
+# Copied from R-3.3.1-intel-2016b.eb.
+# Not sure if BLAS/LAPACK are needed/used when compiling R-math, R documentation doesn't specify this
+preconfigopts = 'BLAS_LIBS="$LIBBLAS" LAPACK_LIBS="$LIBLAPACK"'
+
+# Copied from R-3.3.1-intel-2016b.eb.
+# Again, unsure if these affect R-math: R documentation doesn't specify.
+configopts = "--with-lapack --with-blas --with-pic --enable-threads --with-x=no"
+
+# Since we're only installing Rmath anyway, we don't care about R packages.
+configopts += " --with-recommended-packages=no"
+
+dependencies = [
+	('bzip2', '1.0.6'),
+	('cURL', '7.49.1'),
+	('libreadline', '6.3'),
+	('PCRE', '8.38'),
+	('XZ', '5.2.2'),
+	('zlib', '1.2.8')
+	]
+
+moduleclass = 'numlib'
+
+#To build Rmath, docs say you need to execute 'make' in src/nmath/standalone
+prebuildopts = 'cd src/nmath/standalone;'
+preinstallopts = 'cd src/nmath/standalone;'
+sanity_check_paths = {
+	'files': [],
+	'dirs': [('lib','lib64')]
```

Comment:
Please make this more specific?

Reply:
If I do
'files': [('lib/libRmath.a', 'lib64/libRmath.a')]
'files': [('lib/libRmath.so', 'lib64/libRmath.so')]
will that check if libRmath.a and so both exist (they should, with this easyconig), and are either located under lib ór lib64? Or can I not repeat a 'files': several times to do an 'and'?

Reply:
No, you should use a list with two entries, each a tuple, but you'll need to spread it across two lines to avoid hitting the line length limit (and also, we avoid hardcoding `.so`, since on MacOS it's `.dylib`):

```
sanity_check_paths = {
    'files': [('lib/libRmath.a', 'lib64/libRmath.a'),
              ('lib/libRmath.%s' % SHLIB_EXT, 'lib64/libRmath.%s' % SHLIB_EXT)],
    'dirs': [],
}
```